pytest_fixtures/component/rh_cloud.py

```diff
@@ -31,7 +31,7 @@ def organization_ak_setup(rhcloud_sat_host, rhcloud_manifest_org):
         service_level='Self-Support',
         purpose_usage='test-usage',
         purpose_role='test-role',
-        auto_attach=True,
+        auto_attach=False,
```

Comment:
We do still have some tests using `auto_attach=True` when creating activation keys, so I don't think we can remove the parameter entirely. But yes, since the organization in this fixture is provided by the `rhcloud_manifest_org` fixture in this module, and `rh_cloud_manifest_org` is now using `module_sca_manifest`, we should be covered for all the RH Cloud tests using `organization_ak_setup`.